netjson-monitoring.lua

```diff
@@ -253,4 +275,6 @@ if next(interfaces) ~= nil then
     netjson.interfaces = interfaces
 end
 
-print(cjson.encode(netjson))
+-- json.encode converts `/` to `\/` hence the workaround
+devicedata = cjson.encode(netjson):gsub('\\', '')
```

Comment:
Can you provide an example of why this is necessary? My '/' aren't being converted to '\/'

Reply:
**Without it**: `"name":"\/dev\/root","mounted_on":"\/"`

**Expected**: `"name":"/dev/root","mounted_on":"/"`

I mentioned it above in the issue opening comment :sweat_smile:

Reply:
That is very interesting, I'm seeing `/dev/root` in both cases. We need a third person to test it :)

In any case, can you just apply this `gsub('\\', '') to name and mounted_on, instead of the whole json output?

Reply:
I noticed this now, will do it

Reply:
@PabloCastellano, I mean that the `name` and `mounted_on` are both appearing properly if you print them. But this happens (atleast on my VM) when encoding the table to a json. that is `\/` occurs when you do `cjson.encode(netjson)` and before that everything is fine

Reply:
Yeah in my case I don't see any problem running it on a real device. Are you connected to the VM using SSH? It might be a terminal encoding issue, dunno.

Reply:
Yes, I am connected to the VM using SSH. In anycase if someone else doesn't find this issue it can be easily removed (2 lijnes) and it shouldn't be changing anything else since `\` is an escape character which is normally unexpected

Reply:
@nemesisdesign Can you try it and share your output?

Reply:
Since pandafy confirmed that he doesn't have this problem neither in a VM, I assume there is an encoding issue somewhere in your setup. In any case, the recommended way of developing for OpenWISP is using a real device.